### PR TITLE
fix add/delete/delete-all button

### PR DESCRIPTION
**Pull request by alavieille**

### CODE REVIEW

File: ApiBundle/Context/CMSGroupContext.php [hunk 1]
```diff
@@ -16,6 +16,8 @@ class CMSGroupContext
     const SITE = 'site';
     const STATUS = 'status';
     const STATUS_LINKS = 'status_links';
+    const USER_LINKS = 'user_links';
```
Comment (itkg-nanne) [inline ApiBundle/Context/CMSGroupContext.php]: Maybe we could use a unique const LINKS

File: BackofficeBundle/Controller/ContentTypeController.php [hunk 2]
```diff
@@ -76,10 +76,11 @@ public function newAction(Request $request)
                 $documentManager->persist($contentType);
                 $documentManager->flush();
                 $message = $this->get('translator')->trans('open_orchestra_backoffice.form.content_type.creation');
+                $this->get('session')->getFlashBag()->add('success', $message);
```
Comment (itkg-nanne) [inline BackofficeBundle/Controller/ContentTypeController.php]: Maybe create a ticket to get 'FlashBag' in html template.

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Model/ContentType/ContentType.js [hunk 3]
```diff
@@ -15,7 +15,7 @@ class ContentType extends OrchestraModel
             case "read":
                 return Routing.generate('open_orchestra_api_content_type_show', urlParameter);
             case "delete":
-                urlParameter.contentTypeId = this.get('content_type_id');
+                urlParameter.contentTypeId = this.get('id');
```
Comment (itkg-nanne) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Model/ContentType/ContentType.js]: I would prefer to keep this.get('content_type_id') and change idAttribute in model.

File: BackofficeBundle/Controller/ContentTypeController.php (hunk 2 above)
Comment (alavieille) [inline BackofficeBundle/Controller/ContentTypeController.php]: yes

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/ContentType/ContentTypeFormView.js [hunk 4]
```diff
@@ -94,18 +95,18 @@ class ContentTypeFormView extends mix(AbstractFormView).with(FormViewButtonsMixi
...
 
     /**
-     * Delete
-     * @param {event} event
+     * Delete content type
      */
-    _deleteElement(event) {
-        let contentType = new ContentType({'content_type_id': this._contentTypeId});
+    _deleteElement() {
+        if (null === this._contentTypeId) {
```
Comment (itkg-nanne) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/ContentType/ContentTypeFormView.js]: Is this test really useful as delete button appear only in edit context so with this._contentTypeId not null?

File: WorkflowAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/WorkflowProfile/WorkflowProfileRouter.js [hunk 5]
```diff
@@ -61,7 +61,7 @@ class WorkflowProfileRouter extends AbstractWorkflowRouter
         let url = Routing.generate('open_orchestra_workflow_admin_workflow_profile_new');
         this._displayLoader(Application.getRegion('content'));
         FormBuilder.createFormFromUrl(url, (form) => {
-            let newWorkflowProfileFormView = new NewWorkflowProfileFormView({
+            let newWorkflowProfileFormView = new WorkflowProfileFormView({
```
Comment (itkg-nanne) [inline WorkflowAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/WorkflowProfile/WorkflowProfileRouter.js]: Could you rename newWorkflowProfileFormView to workflowProfileFormView to keep same logic as in other router?

**Comment by alavieille** [comment]
@itkg-nanne pr updated

**Comment by itkg-nanne** [comment]
:+1:

**Comment by ngilain** [comment]
:+1:

**Pull request merged**